aerospike_enterprise/tests/test_e2e.py

```diff
@@ -0,0 +1,86 @@
+# (C) Datadog, Inc. 2025-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+import pytest
+
+from datadog_checks.aerospike_enterprise import AerospikeEnterpriseCheck
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .common import (
+    ALL_METRICS,
+    EXPECTED_PROM_EXPORTER_METRICS,
+    EXPECTED_PROM_LATENCIES_METRICS,
+    EXPECTED_PROM_NAMESPACE_METRICS,
+    EXPECTED_PROM_NODE_STATS_METRICS,
+    EXPECTED_PROM_SETS_METRICS,
+    EXPECTED_PROM_SINDEX_METRICS,
+    EXPECTED_PROM_SYSINFO_METRICS,
+    HERE,
+)
+
+pytestmark = [pytest.mark.usefixtures("dd_environment"), pytest.mark.e2e]
```

Comment:
Can you remove the `pytest.mark.e2e` from the `pytestmark`? The `test_openmetricsv2_check` is not an e2e tests as we are not running the agent here (maybe move it to `test_unit`. I am wondering whether we are not recognizing the integration on the agent when running e2e tests in CI because we first find one in which we do not need to run the agent.